openshift/job-template.yaml

```diff
@@ -0,0 +1,144 @@
+                - name: THOTH_CEPH_BUCKET_PREFIX
+                  valueFrom:
+                    configMapKeyRef:
+                      key: ceph-bucket-prefix
+                      name: thoth
+                - name: THOTH_CEPH_KEY_ID
+                  valueFrom:
+                    secretKeyRef:
+                      name: thoth
+                      key: ceph-key-id
+                - name: THOTH_CEPH_SECRET_KEY
+                  valueFrom:
+                    secretKeyRef:
+                      key: ceph-secret-key
+                      name: thoth
+                - name: THOTH_PACKAGE_ANALYZER_SUBCOMMAND
+                  value: "python"
+                - name: THOTH_DIGESTS_FETCHER_DEBUG
+                  value: "${THOTH_DIGESTS_FETCHER_DEBUG}"
+                - name: THOTH_DIGESTS_FETCHER_PACKAGE_NAME
+                  value: "${THOTH_DIGESTS_FETCHER_PACKAGE_NAME}"
+                - name: THOTH_DIGESTS_FETCHER_PACKAGE_VERSION
+                  value: "${THOTH_DIGESTS_FETCHER_PACKAGE_VERSION}"
+                - name: THOTH_DIGESTS_FETCHER_INDEX_URL
+                  value: "${THOTH_DIGESTS_FETCHER_INDEX_URL}"
+                - name: THOTH_ANALYZER_OUTPUT
+                  valueFrom:
+                    configMapKeyRef:
+                      key: package-analysis-result
+                      name: thoth
+              resources:
+                limits:
+                  memory: '2Gi'
+                  cpu: '1000m'
+                requests:
+                  memory: '2Gi'
+                  cpu: '1000m'
+
+parameters:
+  - name: THOTH_PACKAGE_ANALYZER_JOB_ID
+    description: A job id for package analyzer job.
+    displayName: Job identifier
+    required: true
+  - name: THOTH_DIGESTS_FETCHER_DEBUG
+    displayName: Package Analyzer Verbose
+    description: Run the given package analyzer in a verbose mode so developers can debug
+    required: false
+    value: "false"
+  - name: THOTH_DIGESTS_FETCHER_PACKAGE_NAME
+    displayName: Package name
+    description: Package name which should be analyzed.
+    required: true
+    value: "tensorflow"
+  - name: THOTH_DIGESTS_FETCHER_PACKAGE_VERSION
+    displayName: Package version
+    description: Package version which should be analyzed.
+    required: true
+    value: "1.13"
+  - name: THOTH_DIGESTS_FETCHER_INDEX_URL
+    displayName: Python package indexes
+    description: A comma separated list of Python package indexes (package sources) to analyze packages from.
+    required: true
+    value: "https://tensorflow.pypi.thoth-station.ninja/index/fedora29/1.13/jemalloc/simple"
+  - name: THOTH_REGISTRY
+    description: Registry server from where Image is to be pulled
+    displayName: Registry
+    required: true
+    value: 'docker-registry.default.svc:5000'
+  - name: IMAGE_STREAM_PROJECT_NAME
+    description: Project where ImageStream is present
+    displayName: ImageStream Project
+    required: true
+    value: 'thoth-infra-stage'
```

Comment:
Copied from package-extractor, is it OK?

Reply:
Yes, this should be OK.